compute/launching-an-instance/openstack-horizon.rst

```diff
@@ -1,31 +1,66 @@
+          some of the steps below, you might need to do some :doc:`initial configuration </getting-started/launching-an-instance>` first.
+          An SSH-key created or added in the :doc:`/getting-started/managing-your-cloud/cloud-management-portal` is not available in Horizon as
+          they use `API users </getting-started/users.html#api-users>`__ and SSH-keys are owned by the user account and not the project.
+
+- Under **Project**, click **Compute** and then **Instances** in the left-hand side menu.
+
+- Click the **Launch an instance** button in the upper right corner.
+
+.. caution:: If you want to create an instance in the ``europe-se-1b`` availability zone you need to :doc:`create the
+             volume <../../storage/persistent-block-storage/create-volume>` in that availability zone beforehand and in
+             the **Sources** step select **Boot Source** as **Volume**. This is because we do not support attaching volumes
+             across availability zones and the instance and volume must be in the same availability zone.
+
+- Under the **Details** tab in the dialog.
+
+  - Enter a **Instance Name** your instance and optionally provide a description.
+
+  - Select an **Availability Zone**, see :doc:`Regions and Availability Zones <../regions-and-availability-zones>`.
+
+    - We recommend the default ``europe-se-1a`` availability zone if you are creating your first instance or generally
+      if you are not sure you need to use ``europe-se-1b``.
+
+- Under the **Sources** tab in the dialog.
+
+  - Select **Image** as **Boot Source**.
+
+  - Select **Yes** for **Create New Volume** to boot your instance from a volume, select **No** here if you intend to
+    use a NVME based flavor otherwise we always recommend booting from a volume.
+
+  - Select **No** for **Delete Volume on Instance Delete**, if you specifically want the volume deleted when the
+    instance is deleted select **Yes**.
+
+  - Select the :doc:`Image </images/index>` you want to boot your instance with, press the arrow on the right side to select.
+
+- Under the **Flavour** tab in the dialog.
+
+  - Select the :doc:`flavor <../flavors>` you want the instance to have, press the arrow on the right side to select.
+
+- Under the **Networks** tab in the dialog.
+
+  - Select the network you want to connect your instance to by pressing the arrow on the right side to select it. This could be
+    a :doc:`private network </networking/virtual-router/private-network/index>` if you have one or a
+    :doc:`directly attached IP </networking/directly-attached-ips>` in which case the availability zone is in the name of the network.
+    We recommend using a private subnet, for more information see our :doc:`/getting-started/launching-an-instance` guide.
+
+- Under the **Key Pair** tab in the dialog.
+
+  - Select the SSH key you want added to the instance, press the arrow on the right side to select.
+    This is assuming you are running a Linux instance.
```

Comment:
This is mostly relevant to Linux instances?

Reply:
Made it non-OS explicit